### PR TITLE
Extend or remove email a/b tests

### PR DESCRIPTION
**Pull request by lmath**
## What does this change?
* Opinion and bookmarks A/B tests need to be able to run for another two weeks
* Sleevenotes, film-today, fashion-statement, long-read can be removed
* Update aliases for film-today, fashion-statement and long-read

## What is the value of this and can you measure success?
Tests that need to keep running can keep running and those that have enough data will be tidied up.

### CODE REVIEW

File: common/app/model/EmailNewsletters.scala [hunk 1]
```diff
@@ -263,7 +263,6 @@ object EmailNewsletters {
     description = "Sign up to the Guardian Film Today email and we’ll make sure you don’t miss a thing - the day’s insider news and our latest reviews, plus big name interviews and film festival coverage",
     frequency = "Every weekday",
     listId = 1950,
-    aliases = List(3864, 3865),
     tone = Some("feature"),
```
Comment (davidfurey) [inline common/app/model/EmailNewsletters.scala]: 3864 and 3865 have no subscribers

File: common/app/model/EmailNewsletters.scala [hunk 2]
```diff
@@ -227,8 +227,8 @@ object EmailNewsletters {
     teaser = "Get lost in a great story; the Guardian’s award-winning long reads bring you the biggest ideas and the arguments that matter",
     description = "Get lost in a great story. From politics to fashion, international investigations to new thinking, culture to crime - we’ll bring you the biggest ideas and the arguments that matter. Sign up to have the Guardian’s award-winning long reads emailed to you every Saturday morning",
     frequency = "Every Saturday",
-    listId = 3322,
-    aliases = List(3868, 3869, 3890),
+    listId = 3890,
+    aliases = List(3322),
```
Comment (davidfurey) [inline common/app/model/EmailNewsletters.scala]: 3868 and 3869 have no subscribers

File: common/app/model/EmailNewsletters.scala [hunk 3]
```diff
@@ -334,7 +333,6 @@ object EmailNewsletters {
     description = "A weekly hit of style with substance. Smart fashion writing and chic shopping galleries delivered straight to your inbox. Sign up for our Friday email for the best of the week’s fashion brought to you with expertise, humour and irreverence",
     frequency = "Every Monday",
     listId = 105,
-    aliases = List(3862, 3863),
```
Comment (davidfurey) [inline common/app/model/EmailNewsletters.scala]: 3862 and 3863 have no subscribers

**Review by davidfurey — APPROVED** [review]
Looks good to me.  Thanks Leigh-Anne!

File: common/app/conf/switches/ABTestSwitches.scala [hunk 4]
```diff
@@ -111,61 +111,11 @@ trait ABTestSwitches {
...
-    "Assign some of the new sleeve notes subscribers to receive the old email",
-    owners = Seq(Owner.withGithub("lmath")),
-    safeState = Off,
-    sellByDate = new LocalDate(2017, 6, 1),
-    exposeClientSide = true
-  )
-
-  Switch(
-    ABTests,
-    "ab-the-long-read-email-variants",
```
Comment (katebee) [inline common/app/conf/switches/ABTestSwitches.scala]: If we are deleting this test, do we need to also delete the aliases from `EmailNewsletters.scala`, and update the [interactive](https://github.com/guardian/interactive-email-signups/tree/master/the-long-read)?

Comment (lmath) [inline common/app/conf/switches/ABTestSwitches.scala]: Good point, Kate! Do you have time to look at my [PR](https://github.com/guardian/interactive-email-signups/pull/15)?

**Pull request merged**